src/components/layouts/DeletedTaskRedirectPage.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import { UserRole } from '@/app/api/core/types/user'
```

Comment:
This is a simple static UI so there is no use to make the whole thing a client component imo.

After we remove the part to get token from taskboard selector, we are just making this a client component for the button click -> board view routing. We can simply use `Link` for that, or just make the button a client component. What do you think?

Reply:
Fixed in the commit ce0438e80df34cd4076ef0bf16f21af2263a70ce